.github/workflows/dependency-updates-post-processing.yaml

```diff
@@ -69,10 +59,10 @@ jobs:
           # PR instead of just the latest commit.
           declare -r tidy_message='🤖🦾🛠️ go mod tidy & make package-specs'
 
-          declare -r commit_author_name="${{github.event.head_commit.author.name}}"
-          declare -r commit_message="${{github.event.head_commit.message}}"
+          declare -r commit_author_name="${{steps.configure_git.outputs.commit_author_name}}"
+          declare -r commit_message="${{steps.configure_git.outputs.commit_subject}}"
```

Comment:
The `pull_request` event does not contain the `head_commit` resource so we retrieve the information via `git` CLI.